examples/js/loaders/GLTFDRACOLoader.js

```diff
@@ -0,0 +1,364 @@
+
+          } );
+        break;
+
+      default:
+        throw new Error( 'THREE.GLTFDRACOLoader: Unexpected message type.' );
+
+    }
+
+  };
+
+  function getDecoderModule () {
+    if ( decoderModulePromise ) return decoderModulePromise;
+
+    var promise;
+
+    if ( typeof WebAssembly !== 'object' || decoderConfig.type === 'js' ) {
+      // Load with asm.js.
+      self.importScripts( decoderPath + 'draco_decoder.js' );
+      promise = Promise.resolve();
+    } else {
+      // Load with WebAssembly.
+      self.importScripts( decoderPath + 'draco_wasm_wrapper.js' );
+      decoderConfig.wasmBinaryFile = decoderPath + 'draco_decoder.wasm';
+      promise = fetch( decoderConfig.wasmBinaryFile )
+        .then( function ( wasmResponse ) { return wasmResponse.arrayBuffer(); } )
+        .then( function ( wasmBinary ) { decoderConfig.wasmBinary = wasmBinary; } );
+    }
+
+    // Wait for source files, then create and return a decoder.
+    promise = promise.then( function () {
+      return new Promise( function ( resolve ) {
+        decoderConfig.onModuleLoaded = function ( draco ) {
+          // Module is Promise-like. Wrap before resolving to avoid loop.
+          resolve( { draco: draco } );
+        };
+        DracoDecoderModule( decoderConfig );
+      } );
+    } );
+
+    decoderModulePromise = promise;
+    return promise;
+
+  }
+
+  function decode ( rawBuffer, attributeIDs, attributeTypes ) {
+
+    return getDecoderModule().then( function ( module ) {
+
+      var draco = module.draco;
+      var decoder = new draco.Decoder();
+
+      var buffer = new draco.DecoderBuffer();
+      buffer.Init( new Int8Array( rawBuffer ), rawBuffer.byteLength );
+
+      try {
+
+        return this.decodeGeometry( draco, decoder, buffer, attributeIDs, attributeTypes );
+
+      } finally {
+
+        draco.destroy( buffer );
+        draco.destroy( decoder );
+
+      }
+
+    } );
+
+  }
+
+  function decodeGeometry ( draco, decoder, buffer, attributeIDs, attributeTypes ) {
+
+    var dracoGeometry = new draco.Mesh();
+    var decodingStatus = decoder.DecodeBufferToMesh( buffer, dracoGeometry );
+
+    if ( !decodingStatus.ok() || dracoGeometry.ptr == 0 ) {
+
+      throw new Error( 'THREE.GLTFDRACOLoader: Decoding failed: ' + decodingStatus.error_msg() );
+
+    }
+
+    var geometry = { index: null, attributes: [] };
+    
+    var numFaces = dracoGeometry.num_faces();
+    var numPoints = dracoGeometry.num_points();
+    var numAttributes = dracoGeometry.num_attributes();
+
+    // Add attributes of user specified unique id.
+    for (var attributeName in attributeIDs) {
+      var attributeType = self[ attributeTypes[ attributeName ] ];
+      var attributeId = attributeIDs[attributeName];
+      var attribute = decoder.GetAttributeByUniqueId( dracoGeometry, attributeId );
+      geometry.attributes.push( this.decodeAttribute( draco, decoder, dracoGeometry, attributeName, attributeType, attribute ) );
```

Comment:
this should be 
geometry.attributes.push( decodeAttribute( draco, decoder, dracoGeometry, attributeName, attributeType, attribute ) );

Reply:
Fixed, thanks!